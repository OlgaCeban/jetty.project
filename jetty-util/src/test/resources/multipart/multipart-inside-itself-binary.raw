--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0vTU2oHy5jk6_Kpxn2aE9EokEiRGox4eWfAYP8-
Content-Disposition: form-data; name="reporter"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

<user@company.com>
--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0vTU2oHy5jk6_Kpxn2aE9EokEiRGox4eWfAYP8-
Content-Disposition: form-data; name="timestamp"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

2018-03-21T19:00:18+00:00
--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0vTU2oHy5jk6_Kpxn2aE9EokEiRGox4eWfAYP8-
Content-Disposition: form-data; name="comments"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

this also couldn't be parsed
--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0vTU2oHy5jk6_Kpxn2aE9EokEiRGox4eWfAYP8-
Content-Disposition: form-data; name="attachment"
Content-Type: application/octet-stream
Content-Transfer-Encoding: binary

--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0v
Content-Disposition: form-data; name="fruit"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

cherry
--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0v
Content-Disposition: form-data; name="color"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

red
--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0v
Content-Disposition: form-data; name="cost"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

$1.20 USG
--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0v
Content-Disposition: form-data; name="comments"
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit

--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0vTU2oHy5jk6_Kpxn2aE9EokEiRGox4eWfAYP8-
--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0v--

--94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0vTU2oHy5jk6_Kpxn2aE9EokEiRGox4eWfAYP8---
