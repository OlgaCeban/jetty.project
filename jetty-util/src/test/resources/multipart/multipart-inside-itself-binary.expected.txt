Content-Type|multipart/form-data; boundary="94GJ2MW4vpjh92qj-CHjUUIwb9X8Y2LqkU2Yxn0vTU2oHy5jk6_Kpxn2aE9EokEiRGox4eWfAYP8-"
Parts-Count|4
Part-ContainsContents|reporter|<user@company.com>
Part-ContainsContents|timestamp|2018-03-21T19:00:18+00:00
Part-ContainsContents|comments|this also couldn't be parsed
Part-ContainsContents|attachment|cherry